### ISSUE TITLE
Way to relax constrain for prefix length

### ISSUE THREAD

**Issue by inspirit** [lucidrains/perceiver-ar-pytorch#2]
I think we can get away with having learnable null-latents in case we dont have prefix initially

**Comment by lucidrains** [comment]
@inspirit but you need at least one token that outputs a logit

**Comment by lucidrains** [comment]
yeah, or maybe the other way is to randomly curtail the prefix during training, in which case it will generalize on being conditioned from 0 prefix length to the maximum

**Comment by lucidrains** [comment]
@inspirit yeah, maybe i'll just have to push this responsibility to the dataloading

**Comment by ArEnSc** [comment]
@lucidrains I searched in the paper I don't see a prefix length mentioned. I am confused about this prefix length issue, wouldn't you want the prefix length to be the full size of the context window?
